### DISCUSSION TITLE
[SOLVED!] MSB3073 - The command "dotnet" ... exited with code 1.

### DISCUSSION THREAD

**Discussion by mjl9609** [MonoGame/MonoGame#8487] [General]
I tried checking permissions, building the Content.mcgb file multiple times, reinstalling MonoGame, updating Visual Studio, doing "dotnet tool restore", all in brand new projects, to no success.

The specific error message:

Severity    Code    Description    Project    File    Line    Suppression State
Error (active)    MSB3073    The command "dotnet "" mgcb /quiet /@:"C:\Users\mason\Desktop\ps-dsa-mjl9609\PEs\MasonLevine_PE_MonoGameBasics\Content\Content.mgcb" /platform:DesktopGL /outputDir:"C:/Users/mason/Desktop/ps-dsa-mjl9609/PEs/MasonLevine_PE_MonoGameBasics/Content/bin/DesktopGL/Content" /intermediateDir:"C:/Users/mason/Desktop/ps-dsa-mjl9609/PEs/MasonLevine_PE_MonoGameBasics/Content/obj/DesktopGL/net6.0/Content" /workingDir:"C:/Users/mason/Desktop/ps-dsa-mjl9609/PEs/MasonLevine_PE_MonoGameBasics/Content/"" exited with code 1.    MasonLevine_PE_MonoGameBasics    C:\Users\mason\.nuget\packages\monogame.content.builder.task\3.8.1.303\build\MonoGame.Content.Builder.Task.targets    141

Edit: this is an issue I've tried researching, but for some reason, none of the solutions I've found have been helpful. Would love any input whatsoever!

**Comment by KelsamGames** [comment]
If you open Content.mgcb with the content building UI tool (you should be able to just double-click it) and then click the build button in there, you should get some more robust errors.

**Reply by mjl9609** [reply]
There are no errors when building using Content.mgcb. It seems to be a dotnet issue, looking further into it. I'm just unsure why its doing what its doing. Visual Studio 2022 works fine without MonoGame, but anytime I open up a MonoGame specific project, new or existing, somehow it gets errors ONLY when trying to build the project, not the content.

**Reply by KelsamGames** [reply]
By default MG projects, from the last few versions at least, run a task when the project is built that also runs MGCB to build the content. If you check your .csproj you can find the XML near the bottom where this is setup. Perhaps it has a mismatched version or some such?

**Reply by mjl9609** [reply]
something clicked and I found the issue. Essentially all I had to do was clear the Nugent cache by going to **Tools -> NuGet Package Manager -> Package Manager Settings -> _Clear All NuGet Storage_ at the bottom of General.**

**Comment by mjl9609** [comment]
something clicked and I found the issue. Essentially all I had to do was clear the Nugent cache by going to **Tools -> NuGet Package Manager -> Package Manager Settings -> _Clear All NuGet Storage_ at the bottom of General.**

**Comment by davidallenbaer** [comment]
In Visual Studio 2022, Open a developer command prompt by Tools\Command Line\Developer Command Prompt and enter the command "dotnet tool restore".

[a comment by an automated account is not shown]
